server/appointment/upload-documents/uploadDocuments.php

```diff
@@ -263,6 +263,39 @@ function doesClientInformationMatch($clientInformation, $firstName, $lastName, $
 	return $firstNameMatches && $lastNameMatches && $emailAddressMatches && $phoneNumberMatches;
 }
 
+// container/blob naming rules here https://docs.microsoft.com/en-us/rest/api/storageservices/naming-and-referencing-containers--blobs--and-metadata
+function getContainerName($siteId) {
+	$containerName = null;
+	if($siteId == 1) {
+		$containerName = "nebraska-east-union";
+	} else if ($siteId == 2) {
+		$containerName = "victor-e-anderson-library";
+	} else if ($siteId == 3) {
+		$containerName = "jackie-gaughan-multicultural-center";
```

Comment:
Also, by convention, you should use single quotes for "constant" strings in PHP (i.e. for any string you're not doing interpolation with).